openwisp_controller/config/apps.py

```diff
@@ -232,3 +233,12 @@ def register_dashboard_charts(self):
                 'labels': {'': _('undefined')},
             },
         )
+
+    def notification_cache_update(self):
+        from openwisp_notifications.handlers import register_notification_cache_update
+
+        register_notification_cache_update(
+            self.device_model,
+            device_name_changed,
+            signal_uid="config_Device_notification_cache_invalidation",
+        )
```

Comment:
signal_uid was changed to dispatch_uid in the merged PR on openwisp-notifications.

I suggest changing this block to:

```suggestion
        register_notification_cache_update(
            self.device_model,
            device_name_changed,
            dispatch_uid='notification_device_cache_invalidation'
        )
```